doc/api/n-api.md

```diff
@@ -3655,6 +3671,47 @@ object `js_object` using `napi_wrap()` and removes the wrapping. If a finalize
 callback was associated with the wrapping, it will no longer be called when the
 JavaScript object becomes garbage-collected.
 
+### napi_add_finalizer
+<!-- YAML
+added: v8.0.0
```

Comment:
Is v8.0.0 right? Also, should it be marked as experimental, since it needs `NAPI_EXPERIMENTAL`?

Reply:
It does need the indicator that it is experimental. bit confusing that it says N-API version 1 even in the existing docs.https://nodejs.org/docs/latest/api/n-api.html#n_api_napi_add_finalizer. That seems wrong.

Reply:
There are a few like that in the master docs, and some that don't likst the N-API version at all for the ones that are experimental.  Leaving out I think is the right answer.  I'll create a PR to do that in master.